Content.Shared/_RMC14/Medical/IV/SharedIVDripSystem.cs

```diff
@@ -79,30 +79,28 @@ private void OnIVDripAfterHandleState(Entity<IVDripComponent> iv, ref AfterAutoH
         UpdateIVAppearance(iv);
     }
 
-    private void OnIVDripCanDrag(Entity<IVDripComponent> iv, ref CanDragEvent args)
+    private static void OnIVDripCanDrag(Entity<IVDripComponent> iv, ref CanDragEvent args)
```

Comment:
Rider might have suggested this since there's no instance data being used, but by convention we keep all methods that are handling events non-static
```suggestion
    private void OnIVDripCanDrag(Entity<IVDripComponent> iv, ref CanDragEvent args)
```

Reply:
No need to include the ignore comment, Rider shouldn't be showing you that anyways if it's properly importing the project's .editorconfig